client/app/models/pas-form.js

```diff
@@ -45,11 +45,7 @@ export default class PasFormModel extends Model {
 
   @attr('number') dcpProjectblock;
 
-  @attr({
-    defaultValue(pasForm) {
-      return pasForm.package.project.dcpProjectname;
-    },
-  })
+  @attr('string')
```

Comment:
perhaps place it on the same line, like other properties?

Reply:
Woops, yes. Thx